lms/static/sass/discussion/_discussion.scss

```diff
@@ -375,239 +327,94 @@ body.discussion {
-              margin-bottom: 0.2em;
-              font-size: 14px;
-            }
-          }
-
-          .discussion-reply-new {
-            .wmd-input {
-              height: 120px;
-            }
-          }
-
-          // Content that is hidden by default in the inline view
-          .post-extended-content{
-            display: none;
-          }
-        }
+.new-post-article {
+  display: none;
+  margin-top: $baseline;
 
-        .post-tools {
-          box-shadow: 0 1px 1px $shadow inset;
-          background: $gray-l6;
-
-          &:hover {
-            background: #fcfcfc;
-
-            .icon {
-              color: $link-hover;
-            }
-          }
-
-          a {
-            display: block;
-            padding: ($baseline*0.25) $baseline;
-            font-size: 12px;
-            line-height: 30px;
-
-            .icon {
-              color: $link-color;
-              margin-right: ($baseline*0.25);
-            }
-          }
-        }
-      }
-    }
+  .inner-wrapper {
+    max-width: 1180px;
+    min-width: 760px;
+    margin: auto;
   }
 
-  .new-post-article {
-    display: none;
-    margin-top: $baseline;
+  .thread-title {
+    display: block;
+    margin-bottom: $baseline;
+    font-size: $forum-x-large-font-size;
+    color: $gray-d3;
+    font-weight: 700;
+  }
+}
 
-    .inner-wrapper {
-      max-width: 1180px;
-      min-width: 760px;
-      margin: auto;
-    }
+.edit-post-form {
+  @include clearfix();
+  width: 100%;
+  margin-bottom: $baseline;
+  box-sizing: border-box;
 
-    .thread-title {
-      display: block;
-      margin-bottom: $baseline;
-      font-size: 21px;
-      color: $gray-d3;
-      font-weight: 700;
-    }
+  .form-row {
+    margin-top: $baseline;
   }
 
-  section.discussion-pagination {
-    margin-top: ($baseline*1.5);
+  .post-cancel {
+    @include white-button;
+    @include float(left);
+    @include margin($baseline/2, 0, 0, $baseline*0.75);
+  }
 
-    nav.discussion-paginator {
-      float: right;
+  .post-update {
+    @include blue-button;
+    @include float(left);
+    height: 37px;
+    margin-top: ($baseline/2);
+    padding-bottom: 2px;
 
-      ol {
-        li {
-          list-style: none;
-          display: inline-block;
-          padding-right: 0.5em;
-          a {
-            @include white-button;
-          }
-          &.current-page span {
-            display: inline-block;
-            height: 35px;
-            padding: 0 ($baseline*0.75);
-            border: 1px solid $forum-color-border;
-            border-radius: 3px;
-            font-size: 13px;
-            font-weight: 700;
-            line-height: 32px;
-            color: $gray-d3;
-            text-shadow: 0 1px 0 rgba(255, 255, 255, 0.6);
-          }
-        }
-      }
+    &:hover, &:focus {
+      border-color: #222;
     }
   }
 
-  .edit-post-form {
+  .edit-post-title {
     width: 100%;
-    margin-bottom: $baseline;
-    @include clearfix();
+    height: 40px;
```

Comment:
old, from [here](https://github.com/edx/edx-platform/pull/12771/files#diff-8265a803093b14cd0347a6e10ac38548L593) (github diff is wrong)